docs/setting-up/server/helm.md

```diff
@@ -13,32 +10,78 @@
 !!! note alert alert-primary ""
     Helm v3 is needed to run the following steps.
 
+Please refer to the [Kubernetes Supported versions](https://kubernetes.io/releases/version-skew-policy/#supported-versions) and [Helm Version Support Policy](https://helm.sh/docs/topics/version_skew/) to find out supported versions.
+
+PMM should be platform agnostic, but so far, it requires escalated privileges inside of a container. `root` user inside the PMM container is needed. Thus PMM would not work for Kubernetes Platforms such as OpenShift or others that have hardened Security Context Constraints, for example:
+
+- https://docs.openshift.com/container-platform/latest/security/container_security/security-platform.html#security-deployment-sccs_security-platform
+- https://docs.openshift.com/container-platform/latest/authentication/managing-security-context-constraints.html
+
+Kubernetes platforms offer a different set of capabilities. To use PMM in production, you would need backups and, thus storage driver that supports snapshots. Please refer to your Kubernetes and Cloud provider for storage capabilities.
+
+## Locality and Availability
+
+It is not recommended to run the PMM monitoring server on the same system together with monitored database clusters and services.
+
+Please ensure proper locality either by physically separating workloads in Kubernetes clusters or running separate Kubernetes clusters for the databases and monitoring workloads.
+
+Physical separation of workloads can be done with proper Kubernetes nodes configuration, affinity rules, label selections and etc.
+
+Please also ensure the Kubernetes cluster has [high availability](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/ha-topology/) so that in a case of a node failure, the monitoring service will be running and capturing needed data.
```

Comment:
```suggestion
Also, ensure that the Kubernetes cluster has [high availability](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/ha-topology/) so that in case of a node failure, the monitoring service will be running and capturing the required data.
```